grails-app/assets/javascripts/utils/FormValidationsController.js

```diff
@@ -109,133 +206,90 @@ function FormValidationsController() {
       setErrorFor(emailReference, "O email é obrigatório");
       return;
     }
-    if (validateFormatEmail(emailValue) == false) {
-      setErrorFor(emailReference, "Formato incorreto");
+    if (validateFormatEmail(emailValue) != true) {
+      setErrorFor(emailReference, "Formato de e-mail incorreto.");
       return;
     }
     setSucessFor(emailReference);
   }
 
-  function validateFormatEmail(email) {
-    return formatEmail.test(email);
-  }
-
-  function validatePostal() {
-    let postalCodeValue = postalCodeReference.value;
-    if (!postalCodeValue || postalCodeValue.length != correctPostalCodeLength) {
-      setErrorFor(postalCodeReference, "Favor verificar o CEP");
+  function validateCpfValue() {
+    let cpfCnpjValue = cpfCnpjReference.value;
+    cpfCnpjValue = validateFormatCpf(cpfCnpjValue);
+    if (validateFormatCpf(cpfCnpjValue) == false) {
```

Comment:
```suggestion
    if (!validateFormatCpf(cpfCnpjValue)) {
```
Validar outros pontos do PR